tutorial/practicals/LFRic/building_code/4_psydata/README.md

```diff
@@ -267,8 +267,8 @@ like this will be printed:
 
```

Comment:
The title of this section needs to be updated and the intro text updated too.

Reply:
Done.